playbooks/common/openshift-cluster/config.yml

```diff
@@ -17,6 +17,24 @@
       - package_version
       - docker_image_availability
       - docker_storage
+    when: ansible_distribution != "Fedora"
+
+- name: Verify Requirements - Fedora
+  hosts: oo_all_hosts
+  roles:
+  - openshift_health_checker
+  vars:
+  - r_openshift_health_checker_playbook_context: install
+  post_tasks:
+  - action: openshift_health_check
+    args:
+      checks:
+      - disk_availability
+      - memory_availability
+      - package_availability
+      - docker_image_availability
+      - docker_storage
+    when: ansible_distribution != "Fedora"
```

Comment:
@sdodson - yeah, I think that makes sense. Folks running upstream bits "hot off the press" are going to have a higher potential to have versions outside the known test matrix. :+1: 

If we're waiting for others to chime in I'll hold off on a change, otherwise I'll get a fix in. Just let me know.